tests/e2e/specs/out-of-game/home.spec.js

```diff
@@ -306,6 +314,119 @@ describe('Home - Game List', () => {
         cy.get(`[data-cy-spectate-game=${gameId}]`).click();
       });
     });
+
+    it('Disables spectate button if on home view before game finishes', () => {
+      cy.signupOpponent(playerOne.username, playerOne.password);
+      cy.signupOpponent(playerTwo.username, playerTwo.password);
+
+      //Navigate to homepage and select spectate tab
```

Comment:
Nitpick: put a space before the comment for all the comments (there are more in this file, but I'll just comment it here)
```suggestion
      // Navigate to homepage and select spectate tab
```